lib/Froxlor/Api/Commands/FpmDaemons.php

```diff
@@ -261,6 +266,8 @@ public function add()
 	 *        	optional, default 0
 	 * @param string $limit_extensions
 	 *        	optional, limit execution to the following extensions, default '.php'
+	 * @param string $custom_config
+	 *        	optional, custom settings appended to phpfpm configuration
```

Comment:
this is not appended to the phpfpm configuration but rather to the pool-configuration (not that people get confused and think this would be appended to the `php-fpm.conf` file). Just cosmetics though